scripts/install_theme_vendor.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash -e
+
+VENDOR_DIR="mkdocs-material/material/assets/vendor"
+rm -rf "$VENDOR_DIR"
+
+# MathJax
+TEMP_DIR="$(mktemp -d -t "download-mathjax-XXXXXXXX")"
+MATHJAX_URL="https://registry.npmjs.org/mathjax/-/mathjax-2.7.5.tgz"
```

Comment:
I think it should configurable and have a default value, like https://github.com/OI-wiki/OI-wiki/blob/1f55038fc39b85e8450b202fc35364a2caf4a6e1/Dockerfile#L13, but use `${varname:=default}`
```suggestion
${MATHJAX_URL:="https://registry.npmjs.org/mathjax/-/mathjax-2.7.5.tgz"}
```

Reply:
Sure. And maybe we should document the variables somewhere?

Reply:
Should `OI-wiki/mkdocs-material` and `OI-wiki/gitalk` also be configurable?

Reply:
Okay, the documentation is only in README.md. I grep the'install_theme' keyword, which can only be found here.

Reply:
> Should `OI-wiki/mkdocs-material` and `OI-wiki/gitalk` also be configurable?

Yes, it may be difficult for someone to connect to GitHub

(PS: GitHub does not automatically update the page is really a wrong design)